packages/components/psammead-story-promo/README.md

```diff
@@ -41,10 +45,59 @@ The `LiveLabel` component is to be used inside a `Link` in index pages to show a
 ### Props
 
 <!-- prettier-ignore -->
-| Argument  | Type | Required | Default | Example |
-| --------- | ---- | -------- | ------- | ------- |
-| service | string | yes | N/A | `'news'` |
-| dir | string | no | `'ltr'`| `'rtl'` |
+| Argument | Type   | Required | Default | Example  |
+| -------- | ------ | -------- | ------- | -------- |
+| service  | string | yes      | N/A     | `'news'` |
+| dir      | string | no       | `'ltr'` | `'rtl'`  |
+
+## IndexAlsos
+
+The Index Alsos are links to stories that are related to the top story.
+
+Within the `IndexAlsos` component there is a Visually hidden level 4 which announces text passed as prop.
```

Comment:
what's the convention here? are we capitalising component names? If so, it should say `Visually Hidden`. 

Also, level 4? Level 4 heading? Feels like there is something missing in this sentence.